src/pages/Home.tsx

```diff
@@ -1,10 +1,86 @@
 import Container from '@mui/material/Container';
+import { useEffect } from 'react';
+import Typography from '@mui/material/Typography';
+import Box from '@mui/material/Box';
+import { getAccounts } from '../api/account/AccountApi';
+import { useAppDispatch, useAppSelector } from '../redux/hooks';
+import { loadAccounts, unLoadAccount } from '../redux/slices/accountSlice';
+import AccountCard from '../components/AccountCard';
+import TransactionList from '../components/TransactionList';
 
 function Home() {
+  const dispatch = useAppDispatch();
+  const accounts = useAppSelector((state) => state.account);
+
+  useEffect(() => {
+    (async () => {
+      try {
+        const acc = await getAccounts();
+        dispatch(loadAccounts(acc));
+      } catch (e: any) {
+        dispatch(unLoadAccount());
+      }
+    })();
```

Comment:
Está buena ésta implementación de llamar una función asíncrona dentro del useEffect.
Me gustaría, si no te molesta, llamarlo de ésta manera:
```suggestion
const getAccounts = async () => { ... }; // Definirla
getAccounts(); // Llamarla, dentro del useEffect 
```